csFastFloat/FastDoubleParser.cs

```diff
@@ -110,6 +110,41 @@ unsafe static internal Double ParseNumber (char* first, char* last, chars_format
       return ToFloat(pns.negative, am);
     }
    
+    unsafe static internal Double ParseNumber (byte* first, byte* last, chars_format expectedFormat = chars_format.is_general, char decimal_separator = '.')
+    {
+      while ((first != last) && Utils.is_space(*first))
+      {
+        first++;
+      }
+      if (first == last)
+      {
+        throw new ArgumentException();
```

Comment:
We started the discussion about "throw-helpers" after this draft-PR was started, but keep this in mind if this PR should be merged.

Reply:
Done.